ballot_box/views.py

```diff
@@ -169,6 +169,20 @@ def login(redirect_after=None):
         return force_auth()
 
 
+@app.route("/logout/")
+def logout():
+    if g.get("user") is None:
+        conn_id = session.get("conn_id", False)
+        conn_token = session.get("conn_token", False)
+
+        if conn_id and conn_token:
+            db.session.query(Connection).filter_by(
+                id=conn_id, token=conn_token).delete()
```

Comment:
Bohuzel `Connection` se vyuziva take jako "log" prihlaseni, takze bych to nemazal, ale bud nejak zmenil token, nebo pridal pole pro zneplatneni.